internal/auth0/client/resource.go

```diff
@@ -116,6 +116,28 @@ func NewResource() *schema.Resource {
 				Optional:    true,
 				Description: "Set of URLs that are valid to call back from Auth0 for OIDC backchannel logout. Currently only one URL is allowed.",
 			},
+			"oidc_backchannel_logout_initiators_mode": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"all", "custom",
+				}, false),
+				Description: "Which initiators should receive a logout token. Possible values are `all` indicating all " +
+					"initiators or `custom` meaning only those selected via oidc_backchannel_logout_initiators",
+			},
+			"oidc_backchannel_logout_initiators": {
+				Type: schema.TypeSet,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+					ValidateFunc: validation.StringInSlice([]string{
+						"rp-logout", "idp-logout", "password-changed", "session-expired", "session-revoked",
+						"account-deleted", "email-identifier-changed", "mfa-phone-unenrolled", "account-deactivated",
+					}, false),
+				},
+				Optional: true,
+				Description: "Which specific initiators should receive a logout token. Possible values are " +
+					"rp-logout, idp-logout, password-changed, session-expired, session-revoked, account-deleted, email-identifier-changed, mfa-phone-unenrolled, account-deactivated",
+			},
```

Comment:
Perhaps it would be better to create a nested `OIDCLogout` structure instead of the current approach, as `oidc_backchannel_logout_urls` already contains other values. We can deprecate `oidc_backchannel_logout_urls` and introduce `OIDCLogout`, with nested properties such as `oidc_backchannel_logout_urls` and `BackChannelLogoutInitiators`, similar to the implementation in the SDK.